Gemfile

```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+ruby File.read(".ruby-version").chomp
```

Comment:
I'm in two minds about this. On the one hand, it's DRYer, but on the other hand since we're using explicit versions for everything else in the gem file it feels like it breaks the norm.

(Not to say that because we don't do it elsewhere we shouldn't start here, just that explicit ruby version in the gemfile is the accepted idiom).